src/js/pyodide.ts

```diff
@@ -202,13 +218,17 @@ export async function loadPyodide(
     _node_mounts: [],
     env: {},
     packageCacheDir: indexURL,
+    _pythonInspect: 1,
     packages: [],
     enableRunUntilComplete: false,
   };
   const config = Object.assign(default_config, options) as ConfigType;
   if (!config.env.HOME) {
     config.env.HOME = "/home/pyodide";
   }
+  if (config._pythonInspect) {
+    config.env.PYTHONINSPECT = "1";
```

Comment:
How about using `PYTHONINSPECT` env variable directly instead of using a separate `_pythonInspect` parameter?

For instance:

```
config.env.PYTHONINSPECT = config.env.PYTHONINSPECT ?? "1"
```

Reply:
Sounds good.

Reply:
This will make the CLI runner handle the user setting `PYTHONINSPECT` too.